src/components/TaggablePhoto/TaggablePhoto.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Card, Dropdown, Image } from 'semantic-ui-react';
+import { fetchTaggedUsers } from '../../utils/api';
 
 /* TO BE REPLACED BY REAL STUFF LATER */
 const taggedUsers = [{ name: 'Sara Rubin', email: 'sara.rubin@example.com' }];
```

Comment:
If the actual implementation is up and running, would love it if you could delete this 😘